torch/fx/experimental/normalize.py

```diff
@@ -44,27 +47,40 @@ def call_function(self, target : Target, args : Tuple[Argument, ...], kwargs : D
                     return super().call_function(target, args, kwargs)
                 target_for_analysis = if_true
 
-            new_kwargs = self._args_kwargs_to_normalized_kwargs(target_for_analysis, args, kwargs)
-
-            if new_kwargs:
-                # FIXME: `target(**kwargs)` doesn't keep things specified as kwargs
-                # in kwargs
-                return self.tracer.create_proxy('call_function', target, (), new_kwargs)
-
-        return super().call_function(target, args, kwargs)
+            assert callable(target_for_analysis)
+            sig = inspect.signature(inspect.unwrap(target_for_analysis))
+            new_kwargs = self._args_kwargs_to_normalized_kwargs(sig, args, kwargs)
+        else:
+            assert callable(target)
+            torch_op_schemas = get_signature_for_torch_op(target)
+            if torch_op_schemas:
+                for candidate_signature in torch_op_schemas:
```

Comment:
Perhaps add a comment here about what this is doing (i.e. `iterates through possible overloads to determine which one matches`).

Btw, is it possible that there are multiple overloads that this can bind to?

Reply:
> Btw, is it possible that there are multiple overloads that this can bind to?

Yes, but AFAIK the signatures should be in the same order as in overload resolution, which uses the first one that matches the given arguments